debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/parser/LogMinerDmlParser.java

```diff
@@ -276,8 +276,24 @@ private int parseColumnValuesClause(String sql, int start, String[] columnNames,
         index += VALUES_LENGTH;
 
         int columnIndex = 0;
+        StringBuilder collectedValue = null;
         for (; index < sql.length(); ++index) {
+            char lookAhead = (index + 1 < sql.length()) ? sql.charAt(index + 1) : 0;
```

Comment:
This is used, is this intended?

Reply:
My fault, looks like this one introduced in some intermediary version and later somehow survived the refactoring.